administration/src/bp-modules/applications/NoteDialogController.tsx

```diff
@@ -67,7 +68,12 @@ const NoteDialogController = ({
   return (
     <>
       <Tooltip content={toolTipContent}>
-        <NoteButton onClick={() => onOpenNoteDialog(true)} intent='none' icon='annotation'>
+        <NoteButton
```

Comment:
🙃 I'm not super happy with the styling of the button. Its gray and onHover white.
Not sure if this is intended in the design but seems weird to me since a hover should be highlighting.
But maybe thats just a personal preference

Reply:
I agree, did not test this thoroughly enough.

The MUI palette system is a bit lacking in view, too many common cases are not covered, for example, there is no configurable way for inverted panels, for example light text on dark background in a light theme. It can be fixed with module augmentation as shown in `src/types/mui-theme.d.ts` though.

I would like an official style guide here.